var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -62,6 +64,14 @@ class NetcdfC(AutotoolsPackage):
         when="@4.7.2",
     )
 
+    patch("4.8.1-win-hdf5-with-zlib.patch", when="@4.8.1: platform=windows")
+
+    patch("netcdfc-win-inc-mpi.patch", when="platform=windows")
```

Comment:
I think this and the next two patches should be combined: are the upstream changes these patches cover being managed as one PR? If not, then combining them could potentially lead to conflicts; I assume they'd all make it into the same release though.

Reply:
This patch in particular should not be folded into the other changes as it was not included in my upstream PR for reasons mentioned in the other comment (I've found several threads comprised of debate as to whether this is hdf5's issue or netcdfc's). That just leaves the two below, which I'll try to combine into one patch.

Reply:
Oh sorry, this comment was pinned under the wrong line in the UI. Disregard, my comment, all three should be in one patch.

Reply:
All patches have been condensed.